bot/src/main/java/com/dongtronic/diabot/platforms/discord/commands/nightscout/NightscoutApplicationCommand.kt

```diff
@@ -186,23 +186,29 @@ class NightscoutApplicationCommand : ApplicationCommand {
     }
 
     private fun getUrl(event: SlashCommandInteractionEvent) {
-        NightscoutFacade.getUser(event.user).subscribe {
+        val errorMessage = "You do not have a configured Nightscout URL. Use `/nightscout set url` to configure it."
+        NightscoutFacade.getUser(event.user).subscribe({
             if (it.url != null) {
                 event.reply("Your configured Nightscout URL is `${it.url}`").setEphemeral(true).queue()
             } else {
-                event.reply("You do not have a configured Nightscout URL. Use `/nightscout set url` to configure it.").setEphemeral(true).queue()
+                event.reply(errorMessage).setEphemeral(true).queue()
             }
-        }
+        }, {
+            event.reply(errorMessage).setEphemeral(true).queue()
```

Comment:
Please inspect which exception is thrown instead of assuming that the data is missing.